tools/src/merger/OpenApiMerger.ts

```diff
@@ -85,7 +85,9 @@ export default class OpenApiMerger {
       const spec = read_yaml(`${folder}/${file}`)
       const category = file.split('.yaml')[0]
       this.redirect_refs_in_schema(category, spec)
-      this.schemas[category] = spec.components.schemas as Record<string, OpenAPIV3.SchemaObject>
+      if (spec.components?.schemas !== undefined) {
+        this.schemas[category] = spec.components?.schemas
+      }
```

Comment:
Is this change to make it easier to test? Because `spec.components.schemas` always exists in the actual spec. The Merger runs on the assumption that the Spec has past the Linter check.

Reply:
Yes, the spec I added in https://github.com/dblock/opensearch-api-specification/tree/fail-on-additional-properties/tools/tests/tester/fixtures/specs/complete doesn't have this. It feels like it should be possible to have a valid spec without schemas, no?

Reply:
An OpenAPI spec, yes. But not an OpenSearch OpenAPI Spec. I'm just nit picking. This bit of code is harmless to the main use of the merger.